examples/data-sources/b1ddi_dns_auth_nsgs/data-source.tf

```diff
@@ -1,18 +1,24 @@
 terraform {
   required_providers {
     b1ddi = {
-      version = "0.1"
       source  = "infobloxopen/b1ddi"
     }
   }
 }
 
 # Get DNS Auth NSG with the specified name
-data "b1ddi_dns_auth_nsgs" "tf_example_auth_nsg" {
+data "b1ddi_dns_auth_nsgs" "tf_example_auth_nsgs" {
   filters = {
-    name = "tf_example_auth_nsg"
+    name = "tf_example_auth_nsgs"
```

Comment:
Is `tf_acc_test_auth_nsgs` supposed to match `tf_acc_test_auth_nsg` defined in `b1ddi/resource_dns_auth_nsg_test.go`?

Reply:
Reverted it back to `tf_acc_test_auth_nsg`